contracts/prize-strategy/PrizeStrategy.sol

```diff
@@ -32,6 +32,7 @@ contract PrizeStrategy is PrizeStrategyStorage,
 
```

Comment:
This contract is still marked "IERC777Recipient" - is this correct?

Reply:
Eventually, yes.  It is still there because we need to implement the V2 -> V3 migration code.